src/stores/pageStore.ts

```diff
@@ -0,0 +1,43 @@
+import { makeObservable, observable, action, runInAction } from 'mobx';
+import axios from 'axios';
+import { apiBase } from '../config/api';
+import get from 'lodash/get';
+
+interface IContent {
+  page_type: string;
+  title: string;
+  content: {
+    introduction: {
+      copy: string
```

Comment:
The copy node within each section of the page entity will be an array of strings with 1 or more members, e.g. introduction.copy has 1 but about.copy has 2.